src/test/java/com/microsoft/sqlserver/jdbc/databasemetadata/DatabaseMetaDataTest.java

```diff
@@ -592,7 +604,7 @@ public void testGetMaxConnections() throws SQLException {
     public static void setupTable() throws SQLException {
         try (Statement stmt = connection.createStatement()) {
             stmt.execute("CREATE TABLE " + AbstractSQLGenerator.escapeIdentifier(tableName)
-                    + " (col1 int NOT NULL, col2 varchar(200), col3 decimal(15,2))");
+                    + " (col_1 int NOT NULL, col_2 varchar(200), col_3 decimal(15,2))");
```

Comment:
should we add more test cases other than just "_"?

Reply:
I added more test cases.